auth/secrets.js

```diff
@@ -0,0 +1,5 @@
+
+module.exports = {
+    jwtSecret: process.env.JWT_SECRET || 'keep it secret, keep it safe!',
```

Comment:
perfect, if working out of a non-shared .env file, you always want to include a non-.env fallback